lib/stateTransition/StateTransitionFacade.js

```diff
@@ -93,7 +94,12 @@ class StateTransitionFacade {
       validateIdentityExistence,
     );
 
-    this.createStateTransition = createStateTransitionFactory();
+    this.createStateTransitionFromJSON = createStateTransitionFactory(stateRepository, {
```

Comment:
Honestly, an option in the factory looks strange. I was thinking that we will add it to the function itself.

Reply:
Yeah, mixed up a bit :+1: